packages/blade/src/components/Spinner/_decisions/decisions.md

```diff
@@ -0,0 +1,62 @@
+# Spinner <!-- omit in toc -->
+A spinner is an element with a looping animation that indicates loading is in process.
+
+<img src="./spinner-thumbnail.png" width="150" />
+
+## Design
+- [Figma - Spinner](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=14825%3A203592)
+
+
+## BaseSpinner
+A BaseSpinner will contain some extra props like `intent`. This will not be exposed to the consumer and will only be used for internal components.
+
+### API
+
+| Prop | Type | Default | Description | Required |
+|---|---|---|---|---|
+| accessibilityLabel | `string` | `undefined` | The accessibility label (aria-label) for the spinner. | ✅ |
+| intent | `positive`,`negative`, `notice`, `information`, `neutral` | `undefined` | The intent of the spinner to be rendered. |  |
+| contrast | `low`, `high` | `low` | The contrast of the spinner to be rendered. |  |
+| size | `small`, `medium`, `large` | `medium` | The size of the spinner to be rendered. |  |
+
+
+## Spinner
+A wrapper around Spinner which will be exposed to the consumer to be used as a Spinner. This component will not allow passing `intent` to the Spinner.
+
+### API
+
+| Prop | Type | Default | Description | Required |
+|---|---|---|---|---|
+| accessibilityLabel | `string` | `"Loading"` | The accessibility label (aria-label) for the spinner. |
+| contrast | `low`, `high` | `low` | The contrast of the spinner to be rendered. |  |
+| size | `small`, `medium`, `large` | `medium` | The size of the spinner to be rendered. |  |
```

Comment:
Many design systems allow consumers to change the timing of the animation. Certain user flows may require a slower/faster animation than what we are providing as default. Or have we decided to be rigid when it comes to the timing? Given we have clearly defined motion tokens I think we can entertain timing as a prop  - a question for @saurav12 to pose to Dushyant.